Cargo.toml

```diff
@@ -12,16 +12,14 @@ edition = "2018"
 arrayvec = { version = "0.4", default-features = false, features = ["array-sizes-33-128", "array-sizes-129-255"] }
 serde = { version = "1.0", optional = true }
 parity-scale-codec-derive = { path = "derive", version = "^1.0.2", default-features = false, optional = true }
-bitvec = { version = "0.14.0", default-features = false, features = ["alloc"], optional = true }
+bitvec = { version = "0.15", default-features = false, features = ["alloc"], optional = true }
```

Comment:
this is a breaking change as 0.14 -> 0.15 is breaking change.

Reply:
You mean they changed the internal representation we use to serialize?

Reply:
I havn't looked at this but probably not. My point was just that this change introduce a breaking change in parity-scale-codec.

because a major update of an exposed dependency is a breaking change. user of your libraries will have to make this major update for this dependency as well for their code not to break.

Reply:
indeed that might be difficult to keep up with bitvec and generic-array if we have to release major release each time we want to update them.

But we can probably implement for both version if needed no ? like:
```toml
bitvec15 = { version = "0.15" }
bitvec16 = { version = "0.16" }
```

Reply:
still this